Sources/SwifterSwift/UIKit/UIImageViewExtensions.swift

```diff
@@ -57,6 +57,12 @@ public extension UIImageView {
         imgView.blur(withStyle: style)
         return imgView
     }
+    
+    /// SwifterSwift: Removes the applied blur effect from the image view.
+    func removeBlur(){
+            let blurredEffectViews = self.subviews.filter { $0 is UIVisualEffectView }
+            blurredEffectViews.forEach { $0.removeFromSuperview() }
```

Comment:
Your code seems more readable.

Reply:
I have completed the modification and moved this function to `UIViewExtensions`. However, I am not sure if this is the appropriate location. I placed it there based on my judgment of similar types of functions.

Reply:
My code isn't just more readable, it's also more efficient. Your code creates a new array _and then_ iterates over it. My solution goes over the array just once, skipping elements when required. If you want to do it using functional programming, you need to use the `lazy` keyword.
```swift
subviews
   .lazy
   .filter { $0 is UIVisualEffectView }
   .forEach { $0.removeFromSuperview() }
```
This would have the same performance as my code (basically), as `lazy` creates a view of the array, rather than a copy.

Reply:
Thank you. I will have to write my code with efficiency in mind. I learned a lot.